### DISCUSSION TITLE
Feature suggestions: Restore default frame setting and/or a custom "diff" file.

### DISCUSSION THREAD

**Discussion by PL1-Arcade** [Jakobud/skipstartupframes#10] [Ideas]
1. Would it be possible to modify ssf.txt to add a default value and add a way to reset to default in the menu--either a separate menu entry or pressing ENTER while the frame target menu entry is highlighted?

  Current format for ssf.txt:
[romname],[current frame target value]

  Proposed format for ssf.txt:
[romname],[current frame target value],[default frame target value] **or**
[romname],[default frame target value],[current frame target value]

2. If users make changes to the frame target, those changes will be lost when they update to a new version of the plugin.

Would it be possible to write changes to a separate custom.txt "diff" file that overrides ssf.txt?

If you use this approach, ssf.txt can keep the [romname],[default frame target value] format and write [romname],[current frame target value] to custom.txt on game exit when the value is changed.

New releases of the plugin would not include a custom.txt file so the user's old custom settings would not be overwritten by the new ssf.txt file.

As with the first suggestion, it would be great to have a way to reset frame target to default from the menu.

Scott

**Comment by PL1-Arcade** [comment]
The new v2.0.0 release saves user-changed settings to ssf_custom.txt so they won't be overwritten by a new release.  :+1: 

Would it be possible to reset frame target for the current game to default from the menu, either by deleteing that line from custom.txt or setting it to the value from ssf.txt?


Scott

**Reply by Jakobud** [reply]
I'm not sure if the menu system allows for that. I will investigate.
